pinot-broker/src/main/java/org/apache/pinot/broker/api/resources/PinotClientRequest.java

```diff
@@ -221,6 +230,8 @@ public void processSqlWithMultiStageQueryEngineGet(
   })
   @ManualAuthorization
   public void processSqlWithMultiStageQueryEnginePost(String query, @Suspended AsyncResponse asyncResponse,
+      @ApiParam(value = "Return a cursor instead of complete result set") @QueryParam("getCursor") Boolean getCursor,
```

Comment:
Read like that it looks like no row will be returned. I would say something like: Returns the first `numRows` documents of the result and opens a cursor that can be used to get the rest.

Reply:
Does it need to be a `Boolean`? Couldn't be just a `boolean` that defaults to false?

Reply:
Changed